### PR TITLE
chore: update BPF SDK portions of README

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

File: src/budget-program.js [hunk 1]
```diff
@@ -277,9 +277,7 @@ export class BudgetProgram {
 
       default:
         throw new Error(
-          `A maximum of two conditions are support: ${
-            conditions.length
-          } provided`,
+          `A maximum of two conditions are support: ${conditions.length} provided`,
```
Comment (CriesofCarrots) [inline src/budget-program.js]: "supported" ... might as well fix now?

Comment (CriesofCarrots) [reply src/budget-program.js]: Whoops, too slow

Comment (mvines) [reply src/budget-program.js]: Better luck next time :)

**Pull request merged**